cypress/e2e/tests/migration/controls/businessservices/crud.test.ts

```diff
@@ -17,62 +17,28 @@ limitations under the License.
 
 import { exists, notExists } from "../../../../../utils/utils";
 import { BusinessServices } from "../../../../models/migration/controls/businessservices";
-import { Stakeholders } from "../../../../models/migration/controls/stakeholders";
 
 import * as data from "../../../../../utils/data_utils";
-import { stakeHoldersTable } from "../../../../views/stakeholders.view";
 
-describe(["@tier0", "@interop"], "Business service CRUD operations", () => {
+export function businessServiceCRUD() {
     beforeEach("Interceptors", function () {
-        // Interceptors for business services
         cy.intercept("POST", "/hub/businessservices*").as("postBusinessService");
         cy.intercept("GET", "/hub/businessservices*").as("getBusinessService");
 
-        // Interceptors for stakeholders
         cy.intercept("POST", "/hub/stakeholders*").as("postStakeholder");
         cy.intercept("GET", "/hub/stakeholders*").as("getStakeholders");
     });
 
     it("Business service CRUD", function () {
         const businessService = new BusinessServices(data.getCompanyName(), data.getDescription());
         businessService.create();
-        cy.wait("@postBusinessService");
         exists(businessService.name);
 
         let updatedBusinessServiceName = data.getCompanyName();
         businessService.edit({ name: updatedBusinessServiceName });
-        cy.wait("@getBusinessService");
         exists(updatedBusinessServiceName);
 
         businessService.delete();
-        cy.wait("@getBusinessService");
         notExists(businessService.name);
```

Comment:
Why do we remove the requests waitings? It is a good way of waiting just enough time for the operation to be completed and they're reliable if the network is not stable, the `exists` method just have a timeout, they won't care if the connection is slower than usual.

Reply:
since I moved the method to an export function and the test was passing wihout them , but I can move the interceptors as well in the export to make them work.